modules/ROOT/pages/tutorial/tutorial-sso-configuration.adoc

```diff
@@ -156,91 +156,90 @@ See xref:authentication-authorization/sso-integration.adoc#auth-sso-auth-provide
 
 == Microsoft Entra ID (formerly Azure Active Directory)
 
+The following examples show how to configure Microsoft Entra ID for authentication and authorization using access tokens and ID tokens.
+
+=== Register the application
+
+. Log in to the https://portal.azure.com/#home[Azure portal].
+. Click *Microsoft Entra ID* and navigate to *Manage -> App registrations*.
+. Click *New registration*.
+. Type a name for your application, for example, `Neo4j SSO`.
+. Under *Select the supported account types*, select `Accounts in this organizational directory only (Default Directory only - Single tenant)`.
+. Under *Redirect URI*, select `Single-page application (SPA)` and enter the redirect URI: `http://localhost:7474/browser/?idp_id=azure&auth_flow_step=redirect_uri`
+The redirect URI will accept the returned token responses after successful authentication.
+. Click *Register*.
+
 === Access token
 
-This example shows how to configure Entra ID for authentication and authorization using an access token.
+This example shows how to configure Neo4j to use an Entra ID access token for authentication and authorization.
+
+==== Configure Entra ID
 
-. After the successful creation of your SSO application in Azure, open the *Token configuration* tab to configure a token.
+. From the *App registrations* page, select the app you just created.
+. From the left-hand side menu, navigate to *Manage -> Token configuration*.
 .. Click *Add groups claim*.
 .. Select *Security groups* to include in your access token.
```

Comment:
I'm not sure this is good advice anymore. Perhaps we should suggest `Groups assigned to the application (recommended for large enterprise companies to avoid exceeding the limit on the number of groups a token can emit)`, or just get them to select the option appropriate for them